<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>腾讯地图行政划区</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="keyLineEdit">
      <property name="styleSheet">
       <string notr="true">font-size: 18px;
border: 1px solid #333;
border-radius: 5px;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="secretLineEdit">
      <property name="styleSheet">
       <string notr="true">font-size: 18px;
border: 1px solid #333;
border-radius: 5px;</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pickButton">
      <property name="styleSheet">
       <string notr="true">padding: 10px;
border-radius: 5px;
color: #fff;
background:rgb(0, 170, 255);

QPushButton::hover {
	background:rgb(0, 200, 255);
}</string>
      </property>
      <property name="text">
       <string>获取</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
